src/main/webapp/gmailPanel.js

```diff
@@ -0,0 +1,99 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/* eslint-disable no-unused-vars */
+/* global AuthenticationError, signOut */
+
+// Script for handling the behaviour of the Mail panel's features
+
+let gmailNDays;
+let gmailMHours;
+
+/**
+ * Populate Gmail container with user information
+ */
+function populateGmail() {
+  // Get values for nDays and mHours
+  const nDaysSettingsContainer = document.querySelector('#gmailSettingsNDays');
+  const mHoursSettingsContainer =
+      document.querySelector('#gmailSettingsMHours');
+  gmailNDays = parseInt(nDaysSettingsContainer.innerText);
+  gmailMHours = parseInt(mHoursSettingsContainer.innerText);
+
+  // Get containers for all gmail fields
+  const nDaysContainer = document.querySelector('#gmailNDays');
+  const mHoursContainer = document.querySelector('#gmailMHours');
```

Comment:
I don't know if [this](https://google.github.io/styleguide/htmlcssguide.html#ID_and_Class_Name_Delimiters) applies to HTML IDs as well, I believe these would be better - separated. Same everywhere else if you plan on changing this.

Reply:
Oh interesting I didn't know that was in style guide. I'll fix this everywhere